### DISCUSSION TITLE
Software site location to install BIRT from EWM Client for Eclipse 4.6.x IDE

### DISCUSSION THREAD

**Discussion by ouaggagh** [eclipse-birt/birt#1273] [Q&A]
Hi,

I had a Rational Team Concert (RTC 6.0.6.1) client for eclipse 4.6.x IDE and I was able to install BIRT (and other plugins) from the following URI: http://download.eclipse.org/releases/neon/

Now I have the latest version, called Engineering Workflow Management (EWM 7.0.2) client for Eclipse 4.6.x IDE. I didn't find any documentation that indicate which download site to use. I tried the above link and others (e.g. http://download.eclipse.org/releases/oxygen/, .../photon, .../2022-06, etc.) an none of them works.

Could you please let me know what is the right link to use? I need to install BIRT, and JS plugins.

Thank you

**Comment by speckyspooky** [comment]
The common download-link is: https://download.eclipse.org/birt/downloads/drops/

The latest version is: https://download.eclipse.org/birt/downloads/drops/latest/

There you can download e.g. the all-in-one-designer with "eclipse IDE & BIRT"

**Reply by ouaggagh** [reply]
Thank you.
Actually I don't need to install new eclipse client. I need to use the EWM Client (mentioned above). I tired that common download-link too, but without success. For all the links I added so far in the "Available Software Sites" and selected to install new software I got the "Unable to read repository at http..." error.
As an FYI, the EWM client I have is available here (https://jazz.net/downloads/workflow-management/releases/7.0.2SR1?p=allDownloads) under the Plain .zip Files section: Client for Eclipse 4.6.x IDE (windows platform)

**Reply by wimjongman** [reply]
You are running a very old Eclipse client. you should be able to install it from your normal URL http://download.eclipse.org/releases/neon/ However, you should consult the EWM people if it does not work.
